### PR TITLE
Add setting to adjust font size on desktop app

### PR DESCRIPTION
**Pull request by winstc**
Added a setting to easily adjust the font size in the desktop app.

### CODE REVIEW

**Comment by laurent22** [comment]
Thanks for the pull request, there's just a few things that need to be fixed:
- Please remove the console statements
- Please fix the indentation and use only tabs (not tabs mixed with spaces)
- Let's make it a zoom setting so that it can apply to any theme, and call it `style.zoom` (so that later other style settings can go under the `style` key)

**Comment by winstc** [comment]
I changed ```font_size``` to ```style.zoom```. The default value is now 100 so that it can be used as a percentage scale factor to change the size of other parts of the interface.

File: ElectronClient/app/gui/ConfigScreen.jsx [hunk 1]
```diff
@@ -61,7 +61,7 @@ class ConfigScreenComponent extends React.Component {
 			const settings = Object.assign({}, this.state.settings);
 			settings[key] = value;
 			this.setState({ settings: settings });
-		}
+		};
```
Comment (laurent22) [inline ElectronClient/app/gui/ConfigScreen.jsx]: Please don't add or remove this kind of optional character as it makes the commit needlessly big.

File: ElectronClient/app/gui/ConfigScreen.jsx [hunk 2]
```diff
@@ -101,14 +101,27 @@ class ConfigScreenComponent extends React.Component {
...
 				</div>
 			);
+		} else if (md.type === Setting.TYPE_INT) {
+			const onNumChange = (event) => {
+				const settings = Object.assign({}, this.state.settings);
+				settings[key] = event.target.value;
+				this.setState({ settings: settings});
+			};
+
+			return (
```
Comment (laurent22) [inline ElectronClient/app/gui/ConfigScreen.jsx]: Please make sure only tabs are used and indentation is correct.

Comment (alexdevero) [reply ElectronClient/app/gui/ConfigScreen.jsx]: @winstc Can you please fix the issues with indentation and use tabs?

**Comment by alexdevero** [comment]
@laurent22 since @winstc is not responding, would you mind if I re-create this PR in a new branch that will follow your recommendations?

P.S.: Is Joplin 0.10.47 the latest for desktop?

**Comment by laurent22** [comment]
Fixed in #175

**Pull request closed without merging**